### PR TITLE
Add tooltips to Record Setup and the Play Setup buttons on the Meter Toolbar

### PR DESCRIPTION
**Pull request by abhi-kr-2100**
This implements the minor feature requested by #3968.

Signed-off-by: Abhishek Kumar <abhi.kr.2100@gmail.com>

Resolves: https://github.com/audacity/audacity/issues/3968

Added tooltips to the two setup buttons on the Meter Toolbar.

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: src/toolbars/MeterToolBar.cpp [hunk 1]
```diff
@@ -234,6 +234,7 @@ void MeterToolBar::Populate()
       mRecordSetupButton = safenew AButton(this);
       mRecordSetupButton->SetLabel({});
       mRecordSetupButton->SetName(_("Record Meter"));
+      mRecordSetupButton->SetToolTip(XO("Input Metering"));
```
Comment (vsverchinsky) [inline src/toolbars/MeterToolBar.cpp:237]: Could you please change text to `"Record Meter"` and `"Playback Meter"` respectively? They already have a translation

Comment (abhi-kr-2100) [reply src/toolbars/MeterToolBar.cpp]: Done.

**Review by vsverchinsky — APPROVED** [review]

File: src/toolbars/MeterToolBar.cpp [hunk 2]
```diff
@@ -234,7 +234,7 @@ void MeterToolBar::Populate()
       mRecordSetupButton = safenew AButton(this);
       mRecordSetupButton->SetLabel({});
       mRecordSetupButton->SetName(_("Record Meter"));
-      mRecordSetupButton->SetToolTip(XO("Input Metering"));
+      mRecordSetupButton->SetToolTip(XO("Record Meter"));
```
Comment (vsverchinsky) [inline src/toolbars/MeterToolBar.cpp:237]: Could you please also squash first two commits? That will make commit history clearer

Comment (abhi-kr-2100) [reply src/toolbars/MeterToolBar.cpp:237]: Thank you for pointing this out. I've squashed the first two commits into one.

**Comment by vsverchinsky** [comment]
Yet one thing, please rebase your changes onto release-3.2.2 branch so that we could include your changes into upcoming release

**Comment by abhi-kr-2100** [comment]
@vsverchinsky I used GitHub desktop app to rebase `fix-issue-3968` onto `release-3.2.2`. I've kept all the incoming changes whenever there were conflicts. There were no conflicts on `MeterToolbar.cpp`; the only file my commits change.

**Comment by vsverchinsky** [comment]
You need to drop all commits that came from master branch (via interactive rebase) and force push again, then change target branch of your PR. Alternatively you could create a new branch on top of `release-3.2.2` then cherry pick your commits and create new PR targeting `release-3.2.2`, closing this one.

**Review by vsverchinsky — CHANGES_REQUESTED** [review]
You need to drop all commits that came from master branch (via interactive rebase) and force push again, then change target branch of your PR. Alternatively you could create a new branch on top of release-3.2.2 then cherry pick your commits and create new PR targeting release-3.2.2, closing this one.

**Comment by abhi-kr-2100** [comment]
@vsverchinsky I did a `git rebase -i HEAD~73` (73 being the total number of commits that was previously being shown). I dropped all except the two which were my own. There was a conflict and I chose to keep my changes. I then force pushed and changed the target of PR to `release-3.2.2`.

**Comment by abhi-kr-2100** [comment]
@vsverchinsky Let's scrap this PR. I'll make a new PR and this time target the `release-3.2.2` branch.

**Pull request closed without merging**